### PR TITLE
IntegrationCase is now aware about RuleSet but not Fixers

### PR DESCRIPTION
**Pull request by keradus**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2274#issuecomment-259640406

### CODE REVIEW

File: src/Test/IntegrationCase.php [hunk 1]
```diff
@@ -95,6 +103,10 @@ public function hasInputCode()
         return null !== $this->inputCode;
     }
 
+    public function getConfig()
+    {
+        return $this->config;
+    }
```
Comment (GrahamCampbell) [inline src/Test/IntegrationCase.php]: missing newline

File: src/Test/IntegrationCase.php [hunk 2]
```diff
@@ -125,6 +132,10 @@ public function getRequirements()
         return $this->requirements;
     }
 
+    public function getRuleset()
+    {
+        return $this->ruleset;
+    }
```
Comment (GrahamCampbell) [inline src/Test/IntegrationCase.php]: missing newline

File: src/Test/IntegrationCase.php (hunk 1 above)
Comment (keradus) [inline src/Test/IntegrationCase.php]: great autocommenting Travis bot :D

Comment (keradus) [inline src/Test/IntegrationCase.php]: actually not. see #2287 
with this bug there is priority issue between method_separation and no_extra_blank_lines fixers.

**Pull request closed without merging**